<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ceilingfaan</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSettings"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCtrl_Q">
   <property name="text">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit ceilingfaan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="statusTip">
    <string>Open a tiff stack</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save results</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
